### ISSUE TITLE
[Design - Wireframes] Admin portal - Login/Dash

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/reserve-rec-admin#13]
#### Description:

Create a low-fi mock-up of admin needs. - this ticket is specifically for Login/Dashboard. Additional tickets will be created for other admin sections once the info arch is completed.

#### Acceptance Criteria

- [x] Mockup initial lo-fi designs to include IDIR login
- [ ] Take out initial designs to staff for feedback (note: this could potentially be split into another ticket)
- [ ] Update designs as needed

#### Dependencies

- Blocked by #25 
- Blocking

### Relevant documentation as reference:
[insert links]
[synthesis document/mural]

### Definition of Ready: Design Prototype ticket

- [ ] Acceptance criteria are included
- [ ] relevant documentation to context is linked

### Definition of Done: Design Prototype ticket
- [ ] Designer generates prototype for purposes of user testing

#### PO Review pipeline:
- [ ] Acceptance criteria are checked by Product Owner
- [ ] Approved by Product Owner

**Comment by pavelcreates** [comment]
I think we should separate this out, the tickets being IA and the wireframes separately. I dont think I can do the wireframes on top of all the other stuff I have going on this sprint. IA is definitely doable.

**Comment by pavelcreates** [comment]
I also think it would be beneficial to separate this out by function of the admin panel, which can be informed by the IA we draft. Eg. a ticket to wireframe the reporting feature, a ticket for wireframing the permitting, etc. 

The wireframing ticket, if we split based on the above comment, could be used to wireframe the more high-level portions (ie. the login, dashboard, etc).

**Comment by LindsayMacfarlane** [comment]
@pavelcreates - thank you for that feedback. I agree with your suggestion, so I can pull the IA out of this ticket into a separate one. I'll pull this one out of this sprint.

**Comment by Dianadec** [comment]
This ticket could be more of the login/dashboard umbrella view.
Then, break up this work into different tickets per section of the admin panel. We can do this part once the IA is completed.

**Comment by LindsayMacfarlane** [comment]
@meredithom - this is the ticket I shared with you during our 1:1. You can consider pulling this one in if it helps to work through the architecture with some designs in mind. This ticket focuses on the design of the login and dashboard for admin. 

cc @pavelcreates

**Comment by meredithom** [comment]
Wireframes for Login and Dashboard:

<img width="602" alt="Image" src="https://github.com/user-attachments/assets/4b56b88f-9bc5-4184-bc40-3b0c56513b54" />
[Figma link](https://www.figma.com/design/7ZxtMEp3I425MXcechDB2J/PRDT---Admin?node-id=3044-723)

<img width="420" alt="Image" src="https://github.com/user-attachments/assets/ba8dddfa-8eaf-44a3-9832-1c56a88776cc" />
[Figma link](https://www.figma.com/design/7ZxtMEp3I425MXcechDB2J/PRDT---Admin?node-id=3044-16974)

**Comment by LindsayMacfarlane** [comment]
Thank you for all your work on the login and dashboard @meredithom - I look forward to seeing these implemented :)
